### DISCUSSION TITLE
使用 toml 来当配置序列化实现更好一点

### DISCUSSION THREAD

**Discussion by Gwill** [cpp-main/cpp-tbox#20] [General]
相较于 json ，toml 的表现力更强，时间、日期等都可以直接表示。

**Comment by hevake** [comment]
确实是。

**Comment by hevake** [comment]
出于以下原因选择JSON：
1. JSON很常见，几乎做过开发的人，都会JSON，没有学习难度。
2. JSON在tbox的其它场景有使用，在这里就复用它，不必要引入第三方库；
3. JSON库有merge_patch()功能，能实现参数的合并功能。
